apps/server/src/shared/domain/entity/task.entity.spec.ts

```diff
@@ -873,4 +874,38 @@ describe('Task Entity', () => {
 			expect(schoolId).toEqual(school.id);
 		});
 	});
+
+	describe('getUsersList', () => {
+		describe('when has no users assigned', () => {
+			it('should return an empty array', () => {
+				const task = taskFactory.build();
+
+				expect(task.getUsersList()).toEqual([]);
```

Comment:
just for readability, maybe put the results into a separate variable beforehand (as well for test in line 907)

Reply:
I mean more something like 

`const result = task.getUsersList();
expect (result).toEqual([])`

Reply:
done